mobile-app/app/contexts/FeatureFlagContext.tsx

```diff
@@ -41,14 +41,16 @@ export function FeatureFlagProvider (props: React.PropsWithChildren<any>): JSX.E
   const { network } = useNetworkContext()
   const [retries, setRetries] = useState(0)
 
-  if (isError && retries < MAX_RETRY) {
-    setTimeout(() => {
+  useEffect(() => {
```

Comment:
Why was this moved to useEffect?

Reply:
After the bump of react-redux, it starte throwing infinite loop error

Reply:
For reference: 
<img width="483" alt="image" src="https://user-images.githubusercontent.com/13292662/171137725-ef4ba557-7e06-4db2-9d48-a503b5955ad9.png">